File: python/item/insert_item.py
from item_functions import insert_sample_item, delete_sample_item
from ultracart.exceptions import ApiException

try:
    item_id = insert_sample_item()
    delete_sample_item(item_id)

except ApiException as e:
    print('An ApiException occurred. Please review the following error:')
    print(e)
    exit(1)

File: ruby/item/insert_item.rb
require 'ultracart_api'
require_relative '../constants'
require_relative './item_functions'

begin
  # Create and then delete a sample item
  item_id = insert_sample_item
  delete_sample_item(item_id)

rescue UltracartClient::ApiError => e
  warn 'An ApiException occurred. Please review the following error:'
  p e
  exit(1)
end